external-crates/move/crates/move-unit-test/src/test_reporter.rs

```diff
@@ -61,7 +63,7 @@ pub struct TestStatistics {
     failed: BTreeMap<ModuleId, TestRuns<TestFailure>>,
 }
 
-#[derive(Debug, Clone)]
+// #[derive(Debug, Clone)]
```

Comment:
Why this change?

Reply:
Wasn't needed, so figured I'd remove it. Will remove the comment-out and just delete though.